x/ecocredit/simulation/genesis.go

```diff
@@ -165,92 +191,185 @@ func RandomizedGenState(simState *module.SimulationState) {
 	simState.GenState[ecocredit.ModuleName] = bz
 }
 
+func createClass(ctx context.Context, sStore api.StateStore, class *api.Class) (uint64, error) {
+	cKey, err := sStore.ClassTable().InsertReturningID(ctx, class)
+	if err != nil {
+		return 0, err
+	}
+
+	seq, err := sStore.ClassSequenceTable().Get(ctx, class.CreditTypeAbbrev)
+	if err != nil {
+		if ormerrors.IsNotFound(err) {
+			if err := sStore.ClassSequenceTable().Save(ctx, &api.ClassSequence{
+				CreditTypeAbbrev: class.CreditTypeAbbrev,
+				NextSequence:     2,
+			}); err != nil {
+				return 0, err
+			}
+			return cKey, nil
+		}
+
+		return 0, err
+	} else {
+		if err := sStore.ClassSequenceTable().Update(ctx, &api.ClassSequence{
+			CreditTypeAbbrev: class.CreditTypeAbbrev,
+			NextSequence:     seq.NextSequence + 1,
+		}); err != nil {
+			return 0, err
+		}
+	}
+
+	return cKey, nil
+}
+
+func createProject(ctx context.Context, sStore api.StateStore, project *api.Project) (uint64, error) {
+	pKey, err := sStore.ProjectTable().InsertReturningID(ctx, project)
+	if err != nil {
+		return 0, err
+	}
+
+	seq, err := sStore.ProjectSequenceTable().Get(ctx, project.ClassKey)
+	if err != nil {
+		if ormerrors.IsNotFound(err) {
+			if err := sStore.ProjectSequenceTable().Save(ctx, &api.ProjectSequence{
```

Comment:
Same as above. We are not expecting the sequence table to exist so we should use `Insert`.

```suggestion
			if err := sStore.ProjectSequenceTable().Insert(ctx, &api.ProjectSequence{
```